tests/pyccel/test_pyccel.py

```diff
@@ -882,6 +882,8 @@ def test_classes_f_only( test_file , language):
 @pytest.mark.parametrize( "test_file", ["scripts/classes/classes_2_C.py",
                                         "scripts/classes/classes_5.py",
                                         "scripts/classes/classes_3.py",
+                                        "scripts/classes/classes_1.py",
+                                        "scripts/classes/classes.py",
```

Comment:
If these are handled, do you still need the function `test_classes_f_only`?

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/e0c2cdfe2d24bbbd64c6207157efaa365f2cdb50.